satellite/accounting/live/live_test.go

```diff
@@ -14,35 +14,188 @@ import (
 	"go.uber.org/zap/zaptest"
 	"golang.org/x/sync/errgroup"
 
+	"storj.io/storj/internal/testcontext"
 	"storj.io/storj/internal/testrand"
+	"storj.io/storj/satellite/accounting"
+	"storj.io/storj/storage/redis/redisserver"
 )
 
 func TestPlainMemoryLiveAccounting(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	config := Config{
+		StorageBackend: "plainmemory",
+	}
+	cache, err := NewCache(zaptest.NewLogger(t).Named("live-accounting"), config)
+	require.NoError(t, err)
+
+	// ensure we are using the expected underlying type
+	_, ok := cache.(*plainMemoryLiveAccounting)
+	assert.True(t, ok)
+
+	projectIDs, sum, err := populateCache(ctx, cache)
+	require.NoError(t, err)
+
+	// make sure all of the "projects" got all space updates and got right totals
+	for _, projID := range projectIDs {
+		spaceUsed, err := cache.GetProjectStorageUsage(ctx, projID)
+		require.NoError(t, err)
+		assert.Equalf(t, sum, spaceUsed, "projectID %v", projID)
+	}
+
+	err = cache.ResetTotals(ctx)
+	require.NoError(t, err)
+
+	for _, projID := range projectIDs {
+		spaceUsed, err := cache.GetProjectStorageUsage(ctx, projID)
+		require.NoError(t, err)
+		assert.EqualValues(t, 0, spaceUsed)
+	}
+}
+
+func TestRedisLiveAccounting(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	address, cleanup, err := redisserver.Start()
+	require.NoError(t, err)
+	defer cleanup()
+
+	config := Config{
+		StorageBackend: "redis://" + address + "?db=0",
+	}
+	cache, err := NewCache(zaptest.NewLogger(t).Named("live-accounting"), config)
+	require.NoError(t, err)
+
+	// ensure we are using the expected underlying type
+	_, ok := cache.(*redisLiveAccounting)
+	assert.True(t, ok)
```

Comment:
Why not using [`IsType`](https://godoc.org/github.com/stretchr/testify/assert#IsType)?